.github/workflows/v4-sync.yml

```diff
@@ -0,0 +1,22 @@
+name: v4 sync
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create pull request
+        run: |
+          gh pr create \
+            --base main \
+            --head 1.x \
```

Comment:
Should this be 4.x?

Reply:
Oops :D

Reply:
I changed this to rebase the changes to 4.x branch instead of creating a pull request.